desktop/mac/release-notes/index.md

```diff
@@ -17,6 +17,38 @@ redirect_from:
 
 This page contains information about the new features, improvements, known issues, and bug fixes in Docker Desktop releases.
 
+## Docker Desktop 4.2.0
+2021-11-xx
+
+> Download Docker Desktop
+>
+> [Mac with Intel chip](??: .button .primary-btn }
+> [Mac with Apple chip](??: .button .primary-btn }
+
+### New
+- **Pause/Resume**: Docker Desktop can be paused/resumed. For more information, see [Docker Pause](https://docs.docker.com/go/desktop-pause/).
+
+### Upgrades
+
+- [Docker Engine v20.10.9](https://docs.docker.com/engine/release-notes/#20109)
+- [containerd v1.4.11](https://github.com/containerd/containerd/releases/tag/v1.4.11)
+- [runc v1.0.2](https://github.com/opencontainers/runc/releases/tag/v1.0.2)
+- [Go 1.17.2](https://golang.org/doc/go1.17)
+- [Compose CLI v2.0.1](https://github.com/docker/compose-cli/tree/v2)
+- docker-scan 0.9.0
+
+### Bug fixes and minor changes
+
+- Improved UI: All update related settings are moved to update section and now also available for free tier users.
+- Improved UI: Dashboard window size/position is persited/restored on closing/opening [for-mac#5847](https://github.com/docker/for-mac/issues/5847).
+- Improved: Self-diagnose now also checks for overlap between host IPs and `docker networks`.
+- Removed: Interruptive update popup to inform users about a new Docker Desktop version.
+- Fixed position of update available indicator in dashboard window header.
+- Fixed Docker Desktop sometimes hanging when clicking Exit in the fatal error dialog.
+- Fixed rare startup failure when a `docker volume` is bind-mounted on top of a directory mounted from the host. We now unconditionally remove any `DENY DELETE` ACL entries found on the host directory corresponding to the `docker volume` mountpoint in the VM.
```

Comment:
This needs to be explained from the user pov not the pov of the underlying technology.

Reply:
@djs55 could you help me with that? I think it was your PR.

Reply:
I think the only way to simplify it is to drop information (which we can do)

Here's the info in order of importance:

1. we fixed a rare startup failure
    - we should probably keep this :)
2. the bug only affected you if you had a `docker volume` bind-mounted on top of a directory  mounted from the host
    -  we could drop this, but users might want to know who this bug could have affected / whether this was a bug they experienced
3. the bug fix itself has an observable side-effect: if the user has added their own `DENY DELETE` macOS ACL entries to their files (as they can do if they want), then we will remove those silently.
    - we could drop this, but it's a user-visible negative side-effect of the fix, although hopefully a rare one in practice

Reply:
Thank you @djs55! Your explanation makes a lot of sense and helped me to rephrase the release notes! Maybe double check if it now still contains the message, but I think so.